.github/workflows/linux.yml

```diff
@@ -80,6 +81,20 @@ jobs:
         [ ! -f ${PATCH_PREFIX}.patch ] || patch -p1 --no-backup-if-mismatch < ${PATCH_PREFIX}.patch
         [ ! -f ${PATCH_PREFIX}-${{ matrix.build-type }}.patch ] || patch -p1 --no-backup-if-mismatch < ${PATCH_PREFIX}-${{ matrix.build-type }}.patch
 
+    - name: Set up vcpkg archive cache
+      id: setup-vcpkg-cache
+      run: |
+        mkdir --parents ${{ env.VCPKG_INSTALL_CACHE }}
+        sg lxd -c '/snap/bin/lxc profile device add default vcpkg disk source=${{ env.VCPKG_INSTALL_CACHE }} path=/root/parts/multipass/build/vcpkg_installed'
```

Comment:
```suggestion
        sg lxd -c '/snap/bin/lxc profile device add default vcpkg disk source=${VCPKG_INSTALL_CACHE} path=/root/parts/multipass/build/vcpkg_installed'
```

Reply:
I'm not I understand why the environment variable itself can be used here, but the previous `mkdir` command needs the context expansion?

Reply:
It doesn't need it, I could've updated it, too.

`${{ }}` are processed before it ever gets to BASH.

Reply:
Ok, good to know.  I'll fix these up.

Reply:
Done